pkg/client/config/config.go

```diff
@@ -125,6 +125,8 @@ func loadConfig(context string) (*rest.Config, error) {
 	if len(kubeconfigPath) == 0 {
 		if c, err := loadInClusterConfig(); err == nil {
 			return c, nil
+		} else {
+			log.Error(err, "unable to load in-cluster config")
```

Comment:
It's just a try, for example it may return `ErrNotInCluster` if it is not running in a cluster, so we'd better not log it as an error.

Reply:
@FillZpp Your'e right. I don't know the conventions for logging warnings in this codebase. Are we using WARN or just [INFO,ERROR]? I can change it to info if that's the case. Will this work?
```go
log.Info("unable to load in-cluster config: %s", err.String())
```

Reply:
I would probably use a `defer` that logs this only if `loadConfig` returns an error

Reply:
@alvaroaleman @FillZpp updated the PR accordingly. Now it only logs error when the `loadConfig` method returns an error